src/features/unified-share-modal/__tests__/SharedLinkSection.test.js

```diff
@@ -297,6 +297,78 @@ describe('features/unified-share-modal/SharedLinkSection', () => {
             global.navigator.clipboard = undefined;
         });
 
+        test('should render correct shared link message when permissionLevel is elevated to CAN_EDIT and accessLevel is ANYONE_IN_COMPANY', () => {
+            const sharedLink = {
+                accessLevel: ANYONE_IN_COMPANY,
+                url: 'http://example.com/',
+                isNewSharedLink: false,
+                permissionLevel: CAN_VIEW_DOWNLOAD,
+            };
+
+            const wrapper = getWrapper({ sharedLink });
+
+            expect(wrapper.state().isPermissionElevatedToEdit).toBe(false);
```

Comment:
Should there be introspection into the state here or can we just test the outcome?